app/navigation/nonsipp/LoanMadeOrOutstandingNavigator.scala

```diff
@@ -61,8 +60,9 @@ object LoanMadeOrOutstandingNavigator extends JourneyNavigator {
     case CompanyRecipientNamePage(srn) =>
       controllers.nonsipp.loansmadeoroutstanding.routes.CompanyRecipientCrnController.onPageLoad(srn, NormalMode)
 
-    case page @ CompanyRecipientCrnPage(srn) =>
-      routes.UnauthorisedController.onPageLoad()
+    case CompanyRecipientCrnPage(srn) =>
```

Comment:
Needs the same for partnership route too. Unless it's there but I missed it

Reply:
good spot, added